### ISSUE TITLE
Dragon shedding.

### ISSUE THREAD

**Issue by FireHDn** [AlexModGuy/Ice-and-Fire-Suggestions#1295]
Every time a tamed dragon goes up a stage it will drop it's old scales. The bigger, the more. Once a Fully Grown Stage 5 is reached, it wont produce any scales.

**Comment by CTH999** [comment]
yeah! Seems simple, but effective! One might say it adds some [Personality](https://github.com/AlexModGuy/Ice-and-Fire-Suggestions/issues/1305) to them growing between stages!

maybe as they grow, they will get visibly larger and larger, then shed?
And, maybe the larger dragons shed similar to a lizard, leaving behind a empty husk of skin that you can harvest multiple times for scales, before it's destroyed?
